metadata.py

```diff
@@ -40,13 +40,16 @@ def appmetadata() -> AppMetadata:
         # analyzer_license="",  # short name for a software license
     )
     # and then add I/O specifications: an app must have at least one input and one output
-    metadata.add_input(AnnotationTypes.TimeFrame)
+    metadata.add_input(AnnotationTypes.TimeFrame)  # Output from SWT
+    metadata.add_input(DocumentTypes.TextDocument) # Output from OCR
     metadata.add_output(DocumentTypes.TextDocument)
+    metadata.add_output(AnnotationTypes.Alignment)
+    
     
     # (optional) and finally add runtime parameter specifications
```

Comment:
I think having a parameter to "select" TimeFrame's label would be a good idea. For example, a user can select "credits"-label timeframes only to slice text. Or multiple labels (e.g., chyrons & credit).

Reply:
(probably by an untold convention) we use camelCase for parameter names instead of snake_case. Please rename the parameter name.